src/Authentication/Authenticators/Session.php

```diff
@@ -720,7 +720,7 @@ private function setSessionKey(string $key, $value): void
     /**
      * Remove the key value in Session User Info
      */
-    private function removeSessionKey(string $key): void
+    protected function removeSessionKey(string $key): void
```

Comment:
It also seems the method name is not good.

Reply:
Updated to `removeSessionUserKey`.